enforcer-rules/src/test/java/com/google/cloud/tools/maven/enforcer/LinkageCheckerRuleTest.java

```diff
@@ -23,7 +23,7 @@
 import static org.mockito.Mockito.when;
 
 import com.google.cloud.tools.opensource.dependencies.RepositoryUtility;
-import com.google.cloud.tools.opensource.enforcer.LinkageCheckerRule.DependencySection;
+import com.google.cloud.tools.maven.enforcer.LinkageCheckerRule.DependencySection;
```

Comment:
is the enforcer going to be only available in maven?

Reply:
In addition to the jar containing the enforcer rule, the following artifacts are going to be available in Maven:

- parent pom: `com.google.cloud.tools.maven:cloud-tools-opensource-parent` (used by the enforcer rule)
- the library for analyzing dependency graph: `com.google.cloud.tools.maven:dependencies` (used by the enforcer rule)